core/src/bytes/mod.rs

```diff
@@ -23,6 +23,9 @@ use crate::bytes::trace::NUM_ROWS;
 /// The number of different byte operations.
 pub const NUM_BYTE_OPS: usize = 9;
 
+/// The number of different byte lookup channels.
+pub const NUM_BYTE_LOOKUP_CHANNELS: u32 = 4;
```

Comment:
nit:  Should this be changed to 4 to minimize the increase in columns on the CPU table (via the channel selector columns)?